libs/application/ui-shell/src/components/FormExternalDataProvider.tsx

```diff
@@ -82,29 +79,44 @@ const ProviderItem: FC<{
 
   const errorCode = dataProviderResult?.statusCode ?? 500
   const errorType = errorCode < 500 ? 'warning' : 'error'
-  const { title: errorTitle, summary } = getErrorReasonIfPresent(
-    dataProviderResult?.reason,
-  )
+
+  useEffect(() => {
+    if (dataProviderResult?.reason) {
+      if (Array.isArray(dataProviderResult.reason)) {
+        setReasons(dataProviderResult.reason)
+      } else {
+        setReasons([dataProviderResult.reason as ProviderErrorReason])
+      }
+    }
+  }, [dataProviderResult?.reason, setReasons])
+
+  const getAlertMessage = (reason: ProviderErrorReason) => {
+    const { title: errorTitle, summary } = getErrorReasonIfPresent(reason)
+    return (
+      <AlertMessage
+        type={errorType}
+        title={
+          isTranslationObject(errorTitle)
```

Comment:
Isnt this unnecessary? Pretty sure if a string is provided to formatMessage it returns the string, if a translationobject is supplied it translates the translationObject.

https://github.com/island-is/island.is/blob/main/libs/cms-translations/src/lib/intl.service.ts#L26